python/pysrc/kaskada/_timestream.py

```diff
@@ -269,6 +300,28 @@ def __rsub__(self, lhs: Union[Timestream, Literal]) -> Timestream:
             lhs = Timestream._literal(lhs, self._ffi_expr.session())
         return lhs.sub(self)
 
+    def exp(self) -> Timestream:
+        """
+        Create a Timestreamp raising e to the power of this.
```

Comment:
`e` (put in backticks so it shows up as code)? or italicize to make it look "mathy"